src/components/cart/cart-item/cart-item.module.scss

```diff
@@ -26,9 +30,10 @@
     font: var(--paragraph2);
 }
 
-.remove {
+.removeButton {
     cursor: pointer;
     margin-left: 6px;
+    color: var(--black);
```

Comment:
why did you need to add/change this color?

Reply:
because root element has class `outOfStock` which set color to grey... this includes close button